lc-hax/Scripts/Modules/ESPMod.cs

```diff
@@ -166,6 +166,7 @@ public class ESPMod : MonoBehaviour {
         Action<Color, Vector3>? action,
         float cutOffDistance = 4.0f
     ) {
+        if (bounds == null || camera == null) return;
```

Comment:
Camera isn't nullable here, and same goes for bounds.

Reply:
again, bepinex complained sonething is null there which makes it spam errors.

Reply:
My point is that in this method, `Camera` is typed as non-nullable. A nullable type would be `Camera?`. So it would be best to handle the null check before this function to keep true to the method signature. There's no point if we label something as non-nullable but then check for null. At the very least, we type the references as null. But in this case, like I said earlier, we should handle this before the method.